### DISCUSSION TITLE
Why the "fetch" verb and not something else?

### DISCUSSION THREAD

**Discussion by tom-sherman** [bvaughn/suspense#16] [General]
This is just a random question, but why does the cache use the "fetch" verb and not for example "load" or "read"?

I've seen the latter used before, but fetching feels like it should return a promise to me - in the case of `fetchSuspense` this is not the case ofc.

`readSuspense` and `readAsync` feel better to me, maybe I'm thinking about it wrong.

**Comment by bvaughn** [comment]
There's no objective reason. The only reason I went with "fetch" is that often the method will call the built-in `fetch` API 🤷🏼 

You don't have to stick with the naming terminology though if you don't like it.
```js
import { createCache } from "suspense";

export const {
  fetchAsync: readAsync,
  fetchSuspense: readSuspense,
} = createCache({ ... });
```

**Reply by tom-sherman** [reply]
Cool, just wondering really! I just remember seeing the "read" verb before in the react codebase and examples.

**Comment by bvaughn** [comment]
The longer this thought sits with me, the longer I like "read"

I think I'll rename `fetchSuspense` -> `read` and `fetchAsync` -> `readAsync` in the next version.

**Reply by tom-sherman** [reply]
When this library becomes the jQuery of react, I'm putting it on my CV that I influenced the API 😂
